CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## 0.12.18
 
+* Add an `addTearDown()` function, which allows tests to register additional
+  tear-down callbacks as they're running.
+
```

Comment:
I think it makes the changelog entries more readable, both in markdown and in HTML.